pages/p/[siteId].js

```diff
@@ -0,0 +1,84 @@
+import { useRef, useState } from 'react';
+import { useRouter } from 'next/router';
+import { Box, FormControl, FormLabel, Input, Button } from '@chakra-ui/core';
+
+import Feedback from '@/components/Feedback';
+import { useAuth } from '@/lib/auth';
+import { createFeedback } from '@/lib/db';
+import { getAllFeedback, getAllSites } from '@/lib/db-admin';
+
+export async function getStaticProps(context) {
+  const siteId = context.params.siteId;
+  const { feedback } = await getAllFeedback(siteId);
+
+  return {
+    props: {
+      initialFeedback: feedback
+    }
+  };
+}
+
+export async function getStaticPaths() {
+  const { sites } = await getAllSites();
+  const paths = sites.map((site) => ({
+    params: {
+      siteId: site.id.toString()
+    }
+  }));
+
+  return {
+    paths,
+    fallback: false
+  };
+}
+
+const FeedbackPage = ({ initialFeedback }) => {
+  const auth = useAuth();
```

Comment:
is there any reason why we don't destruct user from auth?
`const { user } = useAuth() `